sdk/src/main/scala/com/horizen/account/forger/AccountForgeMessageBuilder.scala

```diff
@@ -44,15 +44,31 @@ class AccountForgeMessageBuilder(mainchainSynchronizer: MainchainSynchronizer,
   type MS = AccountState
   type MP = AccountMemoryPool
 
-  def computeReceiptRoot(receiptList: Seq[EthereumReceipt]) : Array[Byte] = {
+  def computeReceiptRoot(receiptList: Seq[EthereumConsensusDataReceipt]) : Array[Byte] = {
     // 1. for each receipt item in list rlp encode and append to a new leaf list
     // 2. compute hash
-    TrieHasher.Root(receiptList.map(r => EthereumConsensusDataReceipt.rlpEncode(r.consensusDataReceipt)).toArray)
+    TrieHasher.Root(receiptList.map(EthereumConsensusDataReceipt.rlpEncode).toArray)
```

Comment:
There is this exact piece of code  in AccountBlock at line 51. Could it be possible to put it in a single place in order to avoid code duplication?

Reply:
Ok, done